src/clj/fluree/db/query/fql/parse.cljc

```diff
@@ -145,19 +146,15 @@
 (defn parse-values
   [values context]
   (when values
-    (let [[vars vals] values
-          vars* (keep parse-var-name (util/sequential vars))
-          vals* (mapv util/sequential vals)
-          var-count (count vars*)]
-      (if (every? (fn [binding]
-                    (= (count binding) var-count))
-                  vals*)
-        [vars* (mapv (fn [vals**]
-                       (parse-value-binding vars* vals** context))
-                     vals*)]
-        (throw (ex-info (str "Invalid value binding: "
-                             "number of variables and values don't match: "
-                             values)
+    (let [[vars vals] (policy/normalize-values values)
```

Comment:
If that function will be used for more than just policy related features then it should live in a different namespace.

Reply:
I had it in `parse` originally, but that unfortunately introduced a massive cyclic dependency. I'll see if I can untangle it another way, I agree that this is not the right home for it.

Reply:
I ended up putting it in a new `util` ns, I couldn't really find another place that seemed correct and this seems like the least bad way.